packages/workshop-utils/src/epic-api.server.ts

```diff
@@ -199,7 +226,12 @@ async function getEpicVideoInfo({
 			}
 		},
 	}).catch((e) => {
-		console.error(`Failed to fetch epic video info for ${epicVideoEmbed}`, e)
+		const errorMessage = e instanceof Error ? e.message : String(e)
+		const stackTrace = e instanceof Error ? e.stack : undefined
+		log.error(`failed to fetch epic video info for ${epicVideoEmbed}: ${errorMessage}`, { 
+			videoUrl: epicVideoEmbed,
+			stackTrace 
+		})
```

Comment:
Could this not be more easily done with:


```suggestion
		log.error(`failed to fetch epic video info for ${epicVideoEmbed}:`, e)
```